Apps/WebClient/src/ClientApp/package.json

```diff
@@ -35,46 +34,28 @@
         "babel-preset-env": "^1.7.0",
         "bootstrap": "~4.6.0",
         "compression-webpack-plugin": "^6.1.1",
-        "css-loader": "^5.2.4",
-        "eslint": "^7.25.0",
-        "eslint-config-prettier": "^8.3.0",
+        "eslint": "^6.8.0",
```

Comment:
Why the change to an older version of `eslint`?

Reply:
Apparently, that is the version that is compatible with our other dependencies. Not sure why it was working before, but after clearing the lock file npm was complaining it couldn't find the dependency.